### ISSUE TITLE
Support `tail`ing and `follow`ing logs

### ISSUE THREAD

**Issue by SeMeKh** [kubernetes/dashboard#1083]
`kubectl logs`'s `--tail` and `--follow` options are pretty handy and missing in dashboard. Enabling a sane tail default (e.g. 1000 lines) could also relieve the issue with the browser becoming unresponsive due to huge amount of logs.

**Comment by bryk** [comment]
This is a good feature request. Thanks @SeMeKh for reporting it. We'll take a deeper look at it in our next iteration.

**Comment by steelbrain** [comment]
A friendly bump, this feature would be wonderful to have

**Comment by bryk** [comment]
A general revamp of logs viewing experience is high on our priority list. So we cluster all logs-related issues like this into one big project. We'll let know when this is being taken. 

And yes, please ping the issue if you feel this is taken too slowly.

**Comment by innovia** [comment]
Last comment 10 months ago is this is still in progress?

**Comment by floreks** [comment]
Not until kubernetes API changes. It is very limited when it comes to logs. Right now we could maybe provide proxy link to apiserver endpoint and open raw logs page (pure text) with streaming enabled. We don't really want to do complex logs processing. At least not until other higher priority tasks are finished.

**Comment by cheld** [comment]
--tail option is now implemented internally to avoid out of memory. It is not exposed to the user as option. I don't think it makes sense.

--follow is not implemented. I think a follow should be exposed to the user.

**Comment by maciaszczykm** [comment]
It is already tracked from #1828.
